src/private/files.ts

```diff
@@ -0,0 +1,35 @@
+import { ensureInitialized, sendMessageRequest } from "../internal/internalAPIs";
+import { GlobalVars } from "../internal/globalVars";
+
+/**
+ * Namespace to interact with the files part of the SDK.
+ * This object is used to handle logs
+ *
+ * @private
+ * Hide from docs
+ */
+export namespace files {
+
+  GlobalVars.handlers["log.request"] = handleGetLogRequest;
+
+  export function handleGetLogRequest(): void {
```

Comment:
Does this function need to get exported? As far as I can tell it's internal to just this file and isn't something our developers would even be calling.